src/button-dropdown/item-element/styles.scss

```diff
@@ -23,7 +23,7 @@
   &:first-child {
     margin-top: 0;
   }
-  &.last {
+  &:not(.has-expandable-groups).last {
```

Comment:
This one is supposed to add dividers between groups when not expandable. Because it was always added the dividers were also added to expandable case causing extra border at the bottom of the list.

Reply:
This applies to the elements _inside_ a group, right? It's not like the element itself has a group. Should we use a naming like `.is-expandable-group` or `.in-expandable-group` instead of `.has-expandable-groups`? Same for the `hasExpandableGroups` prop when passed down to the inner level.

Reply:
Approving as I don't consider this critical but would be nice to resolve this to make sure this doesn't confuse us in the future.